docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    import stumpy
</pre><p>It seems the only module that you have used so far from STUMPY API is stump. so, you may want to replace this line with:</p><pre class="ql-syntax" spellcheck="false">from stumpy import stump
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>